R/derive_extreme_event.R

```diff
@@ -613,9 +670,22 @@ event_joined <- function(dataset_name = NULL,
                          join_vars,
                          join_type,
                          first_cond = NULL,
+                         first_cond_lower = NULL,
```

Comment:
Could we cross-link and put the example where `event_joined` is being used in `derive_extreme_event()`

Reply:
Sorry, I am not sure what you mean. Where should I put a link and where should it point to?

Reply:
The example in `derive_extreme_event()` that uses `event_joined`.  Was thinking it could go here as well